packages/scene-composer/src/components/panels/SceneHierarchyPanel/__snapshots__/SceneHierarchyPanel.spec.tsx.snap

```diff
@@ -5,6 +5,15 @@ exports[`SceneHierarchyPanel should render search hierarchy correctly when enabl
   <div
     class="tm-hierarchy-panel"
   >
+    <div
```

Comment:
this test case should be removed

Reply:
This component still uses a feature flag I won't remove which is `SceneHierarchyMultiSelect`. So I should leave the test case for that.